docs/core/preview3/tools/dotnet-add-package.md

```diff
@@ -69,38 +69,38 @@ The package reference to add.
 
 `-h|--help`
 
-Prints out a short help for the command or argument.
+Prints out a short help for the command.
```

Comment:
perhaps it is inferred that `-h` is always available regardless. However, there is not a **dotnet-add.md** and IMO the documentation should reflect it.

Let's not hold this up.